### PR TITLE
Disallow fz files to be used for the reduction

### PR DESCRIPTION
**Pull request by rcooke-ast**
fz files change the extension where the data and header information are loaded, so I've fixed the soar goodman spectrograph file, and added an error for those that include fz files. There is a related PR in the dev suite

### CODE REVIEW

File: pypeit/pypeitsetup.py [hunk 1]
```diff
@@ -25,6 +25,9 @@
 from pypeit.spectrographs.util import load_spectrograph
 
 
+bad_ext = [".fz"]
```
Comment (kbwestfall) [inline pypeit/pypeitsetup.py:28]: Please put this inside the `check_file_list` function.

**Review by kbwestfall — APPROVED** [review]
My only request was to move where you define `bad_ext` to inside the main function.  Other than that, I had the same question as @profxj about the raw format provided by the telescope.  Given that it's just a change in the extension number, it would be nice if this could be something the code handles automatically.  But a quick look at the control flow suggests this is easier said than done...  Along the same lines, I'm a little hesitant to place a blanket ban of the `fz` format for all instruments, but making it specific to `soar_goodman` also seems to be non-trivial.

**Comment by rcooke-ast** [comment]
OK - I'll revert these changes. It seems that they recommend downloading and using `.fz` files. I'll add a note in the docs that only `.fits.fz` files can be used to reduce soar+goodman data.

**Comment by rcooke-ast** [comment]
You're quite right, @kbwestfall - it's not trivial to handle both `.fits.fz` and `.fits`, as I tried doing that first  :-)

**Pull request closed without merging**